packages/create/templates/minimal/src/routes/+page.svelte

```diff
@@ -1,2 +1,10 @@
+<script lang="ts">
+	import favicon from '$lib/assets/favicon.svg';
+</script>
+
+<svelte:head>
+	<link rel="icon" href={favicon} />
+</svelte:head>
```

Comment:
Shouldn't this be part of the layout? As soon as you create a second page, this will behave differently than it does now.

Reply:
Yes, even a better reason to have a layout. 👍